pepper-apis/dsm-core/src/main/java/org/broadinstitute/dsm/jobs/LabelCreationJob.java

```diff
@@ -24,11 +24,13 @@ public void execute(JobExecutionContext context) throws JobExecutionException {
                 if (!kitsLabelTriggered.isEmpty()) {
                     KitUtil.createLabel(kitsLabelTriggered, null);
                 }
-            } else if (labelCreationStarted < System.currentTimeMillis() - (SystemUtil.MILLIS_PER_HOUR * 2)) {
-                logger.error("Label creation job is running for over 2 hours now...");
+            } else if (labelCreationStarted < System.currentTimeMillis() - (SystemUtil.MILLIS_PER_HOUR)) {
```

Comment:
Check out `Instant`--it has really readable ways to do date comparisons with different units, stuff like `Instant.now().minus(1, ChronoUnit.HOURS)`